src/SwooleRequestHandlerRunner.php

```diff
@@ -213,6 +230,21 @@ public function onWorkerStart(SwooleHttpServer $server, int $workerId) : void
             'cwd' => getcwd(),
             'pid' => $workerId,
         ]);
+
+        if ($this->dispatcher) {
+            $this->dispatcher->dispatch(new OnWorkerStartEvent($server, $workerId));
+        }
```

Comment:
Why is this not done in the `onWorkerStart()` method?

Reply:
ah my bad, it's already executed, so there's no need to change with onworkerstart event, but there's no mechanism yet to handle workerstop/workererror event right?

Reply:
I think what i propose is overlapping with an already existing feature like `onWorkerStart()` method in `SwooleHttpServer`, But I'm not try to inject directly `onWorkerStart` event at `SwooleHttpServer `object, instead trigger an event in `SwooleRequestHandlerRunner `object which also follow the `SwooleHttpServer `event.

The `onWorkerStart `event at `SwooleHttpServer ` can't be register multiple event right?